content/zh/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md

```diff
@@ -192,11 +192,11 @@ First, update the following fields in your YAML file:
 
 要在启用了 leader 选举的情况下运行多调度器，你必须执行以下操作：
 
-首先，更新上述 Deployment YAML（my-scheduler.yaml）文件中的以下字段：
+更新 KubeSchedulerConfiguration 中的以下字段 `my-scheduler-config` YAML 文件中的 ConfigMap
 
-* `--leader-elect=true`
-* `--lock-object-namespace=<lock-object-namespace>`
-* `--lock-object-name=<lock-object-name>`
+* `leaderElection.leaderElect`至`true`
+* `leaderElection.resourceNamespace`至`<lock-object-namespace>`
+* `leaderElection.resourceName`至`<lock-object-name>`
 
```

Comment:
Where's the English version of this text? I don't see it. Usually there would be an HTML comment holding the English text.

Reply:
sorry for missing that , I am a beginner and just started contributing to k8s.